tests/MongoDB.Driver.Core.Tests/Core/Authentication/ScramSha1AuthenticatorTests.cs

```diff
@@ -134,20 +135,31 @@ public void Constructor_should_throw_an_ArgumentNullException_when_credential_is
         [Theory]
         [ParameterAttributeData]
         public void Authenticate_should_not_throw_when_authentication_succeeds(
+            [Values(false, true)] bool useLongAuthentication,
```

Comment:
So it turns out we were already testing the short form (2 messages from the driver and 2 replies from the server), so I've enabled testing for the long form.